src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const BASIC_COST_PER_DAY = days * 40;
+
+  const SHORT_RENTAL_TIME = 3;
+  const DISCOUNT_SHORT_TIME = 20;
+
+  const LONG_RENTAL_TIME = 7;
+  const DISCOUNT_LONG_TIME = 50;
+
+  if (days >= LONG_RENTAL_TIME) {
+    return BASIC_COST_PER_DAY - DISCOUNT_LONG_TIME;
+  }
+
+  if (days >= SHORT_RENTAL_TIME) {
+    return BASIC_COST_PER_DAY - DISCOUNT_SHORT_TIME;
+  }
+
+  return BASIC_COST_PER_DAY;
 }
 
 module.exports = calculateRentalCost;
```

Comment:
It's a good practice to export functions at the end of the file, but consider also exporting the constants used for the discount values and rental time thresholds if they could be useful elsewhere in your application.